scout/commands/update/case.py

```diff
@@ -107,7 +107,7 @@ def case(context, case_id, case_name, institute, collaborator, vcf, vcf_sv,
         case_changed = True
 
     if case_changed:
-        adapter.update_case(case_obj)
+        adapter.update_case(case_obj, soft=False)
```

Comment:
What about a more self-explanatory name for this variant: `reset_updated `or something like this?

Reply:
Yes! Will do!